pkg/proxy/nftables/proxier.go

```diff
@@ -1195,12 +1312,12 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 
 		// Capture the clusterIP.
-		tx.Add(&knftables.Element{
+		proxier.clusterIPs.ensureElem(&knftables.Element{
 			Set: clusterIPsSet,
 			Key: []string{svcInfo.ClusterIP().String()},
-		})
+		}, tx)
```

Comment:
I think I'd put `tx` first in this function... it looks weird to have it dangling off the end...

Reply:
haha now I see it too